### ISSUE TITLE
Can't replicate the results

### ISSUE THREAD

**Issue by wxp16** [ArmandDS/bert_for_long_text#2]
Hi, thanks for your work on this.

Now I want to replicate your implementation. Where are the data you used?
I actually downloaded a csv file from Kaggle, but found the total entries is different with your version. 
Cos now the long-document classification model got very bad performance, loss is around 2, 
accuracy is only 0.33. I don't know why. I used the same data pre-processing. Below is the output aftering training 10 epoch.
`Epoch 10/10
4565/4571 [============================>.] - ETA: 0s - loss: 1.8058 - acc: 0.3544Epoch 1/10
 120/4571 [..............................] - ETA: 19s - loss: 1.8386 - acc: 0.3463
Epoch 00010: ReduceLROnPlateau reducing learning rate to 8.573749619245064e-06.
4571/4571 [==============================] - 35s 8ms/step - loss: 1.8057 - acc: 0.3544 - val_loss: 1.8358 - val_acc: 0.3473`

Below is the code for model creation. 
```
def create_keras_model(dimention=None,
                        mask_value=-99.0,
                        lstm_dim=100,
                        dense_dim=30,
                        num_labels=2,
                        learning_rate=2e-5):
    text_input = layers.Input(shape=(None,768,), dtype='float32', name='segments')
    l_mask = layers.Masking(mask_value=mask_value)(text_input)
    encoded_text = layers.LSTM(lstm_dim,)(l_mask)
    out_dense = layers.Dense(dense_dim, activation='relu')(encoded_text)
    out = layers.Dense(num_labels, activation='softmax')(out_dense)
    model = tf.keras.Model(inputs=text_input, outputs=out, name="RoBERT")
    if num_labels==2:
        model.compile(optimizer=tf.keras.optimizers.RMSprop(learning_rate=learning_rate),
              loss='binary_crossentropy',
              metrics=['acc'])
    else:
    
        model.compile(optimizer=tf.keras.optimizers.Adam(learning_rate=learning_rate),
              loss='sparse_categorical_crossentropy',
              metrics=['acc'])  
    return model ```

Please advice and help

**Comment by FL33TW00D** [comment]
@wxp16 
Did you manage to replicate the results?
